crypto/batchverifier.go

```diff
@@ -113,7 +112,7 @@ func (b *BatchVerifier) GetNumberOfEnqueuedSignatures() int {
 // if the batch is zero an appropriate error is return.
 func (b *BatchVerifier) Verify() error {
 	if b.GetNumberOfEnqueuedSignatures() == 0 {
-		return ErrZeroTransactionInBatch
+		return nil
```

Comment:
Yes. This is the intention, so that the logicsig transactions which were not added to BatchVerifier at the level of `TxnGroup` and do not contribute to this `GetNumberOfEnqueuedSignatures`, but were verified at local batches, also get cached.

Reply:
so are we OK this this change?